packages/scans/src/action-helper.js

```diff
@@ -60,7 +60,7 @@ let actionHelper = {
                 if (site.alerts.length !== 0) {
                     msg = `${msg} ${TAB} **New Alerts** ${NXT_LINE}`;
                     site.alerts.forEach((alert) => {
-                        msg = msg + TAB + `${BULLET} **${alert.name}** [${alert.pluginid}] total: ${alert.instances.length}:  ${NXT_LINE}`
+                        msg = msg + TAB + `${BULLET} **${alert.riskdesc}: ${alert.name}** [[${alert.pluginid}]](https://www.zaproxy.org/docs/alerts/${alert.pluginid}/) total: ${alert.instances.length}:  ${NXT_LINE}`
```

Comment:
It's preferable to use `riskcode` and `confidence` and probably better if it's explicit what's what. (That requires mapping the IDs to English though.)

Reply:
@thc202 happy to do that - where can I find documentation on these values and what they map to?

Reply:
(For context: I'm currently working on moving the repo to TypeScript, add tests, and do some other clean up, and would like to either merge the PR very soon, e.g. if there are only few changes that I can do right away, or close this PR and create a new one in the new structure)

Reply:
> @thc202 happy to do that - where can I find documentation on these values and what they map to?

https://www.zaproxy.org/docs/constants/ 😁

Reply:
Thanks, I think that's exactly what I need! :)
I'll close this PR, open the refactoring PR, and will add a new feature PR afterwards.